css/gora.css

```diff
@@ -0,0 +1,540 @@
+body {
```

Comment:
remove all classes like `row2` `column2` etc. and use only one - common class like row, column etc. they are the same. If something is not the same - split into two

e.g. `row2` -> `row` & `section_row`